src/main/kotlin/io/perfometer/http/Http.kt

```diff
@@ -9,11 +9,12 @@ sealed class HttpRequest(val host: String,
                          val body: ByteArray) {
 
     override fun equals(other: Any?): Boolean {
-        if (other !is HttpRequest) {
-            return false
-        }
-        return host == other.host && port == other.port && path == other.path &&
-                headers == other.headers && body.contentEquals(other.body)
+        return other is HttpRequest
```

Comment:
Checking if other is HttpRequest is not needed now.

Reply:
I suggest leaving the check
- it handles null check too
- if the check passes, the compliler does smart casting and there's no need to cast the parameter object to HttpRequest request